packages/plugin-form-builder/src/collections/FormSubmissions/index.ts

```diff
@@ -91,12 +91,12 @@ export const generateSubmissionCollection = (formConfig: PluginConfig): Collecti
       ...(formConfig?.formSubmissionOverrides?.fields || []),
     ],
     hooks: {
+      ...(formConfig?.formSubmissionOverrides?.hooks || {}),
```

Comment:
This change would make it so that if I wanted to add a `beforeValidate` hook, it would completely remove the two `beforeChange` hooks that this plugin provides.

Reply:
Oh its being spread nvm